f3/granite.go

```diff
@@ -4,18 +4,24 @@ import (
 	"bytes"
 	"encoding/binary"
 	"fmt"
-	"sort"
-
 	"github.com/filecoin-project/go-bitfield"
 	rlepluslazy "github.com/filecoin-project/go-bitfield/rle"
 	"golang.org/x/xerrors"
+	"math"
+	"sort"
 )
 
 type GraniteConfig struct {
 	// Initial delay for partial synchrony.
 	Delta float64
 	// Change to delta in each round after the first.
 	DeltaRate float64
```

Comment:
As far as I can tell, this is no longer used in the same way in the new spec. I looks like it only applies after the 5th round.
- Move this down below the other fields, since it takes effect last
- Fix the comment
- Perhaps rename to something more serious, like DeltaBackoffExponent?

Reply:
Done.